### PR TITLE
[terra-clinical-result] Alpha Truncation fixes

### PR DESCRIPTION
**Pull request by BenBoersma**
### Summary
Fixes https://github.com/cerner/terra-clinical/issues/616
Fixes https://github.com/cerner/terra-clinical/issues/619
Fixed alpha truncation issue in flowsheet cell.
Fixed truncation issue with the extra concept displays
Added new truncation tests for base clinical result.
Fixed a misnamed class in Blood Pressure

Please add your name to the [CONTRIBUTORS.md] file. Adding your name to the [CONTRIBUTORS.md] file signifies agreement to all rights and reservations provided by the [License].

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Was it intended to increase the font size for the extra details?
Screen Shot 2020-03-25 at 8 23 34 AM
Screen Shot 2020-03-25 at 8 23 40 AM

**Comment by BenBoersma** [comment]
> Was it intended to adjust the font size for the extra details?

@emilyrohrbough Yes, they weren't correct to begin with. This is because the `datetime-display` wasn't getting that as a class it was getting `concept-display` as its class. 
 These changes https://github.com/cerner/terra-clinical/pull/633/files#diff-13926e711e02256769a4612c0d911268R89

File: packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/clinical-result/TruncatedResult.test.jsx [hunk 1]
```diff
@@ -11,6 +11,18 @@ export default () => (
...
+    <ClinicalResult
+      eventId="111"
+      result={{
+        value: '354.23377777758678567867856785687678567856767868678567856785678567856785678567857',
+        unit: 'mmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHg',
+      }}
+      isTruncated
+      isModified
+      hasComment
+      conceptDisplay={Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral'
```
Comment (mjhenkes) [inline packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/clinical-result/TruncatedResult.test.jsx]: I imagine this should be failing lint? This is what is breaking the heroku deployment.
```suggestion
      conceptDisplay="Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral"
```

**Comment by neilpfeiffer** [comment]
Reviewed: #633
Resolves: #616 (for non IE10 which will be fixed by #619)

|  |
|---|
| UX +1 |

**Comment by ryanthemanuel** [comment]
Main thing to re-review is: https://github.com/cerner/terra-clinical/pull/633/commits/5cb2fca9118f7468d81756ee3ce8227d3e90e3c0

**Comment by PaulSolDev** [comment]
Functional review complete. No issues identified.

**Pull request merged**